filebeat/module/nginx/ingress_controller/_meta/fields.yml

```
@@ -11,6 +11,26 @@
         Real source IP is restored to `source.ip`.
 
     # ingress-controller specific fields
+    - name: upstream_ip_list
+      type: array
```

I think that `type: array` only doesn't generate any mapping. This should be an array of keywords, or just keyword. Elasticsearch doesn't have a data type for arrays, but any field can have multiple values, see [docs about arrays](https://www.elastic.co/guide/en/elasticsearch/reference/7.9/array.html)).

We could use just keyword:
```suggestion
      type: keyword
```

Or if we still want to indicate that this is intended to store multiple values:
```suggestion
      type: array
      object_type: keyword
```

Same for the other lists.